_templates/download.html

```diff
@@ -21,7 +21,9 @@
 <link rel="alternate" type="application/rss+xml" href="/en/rss/releases.rss" title="Bitcoin Core releases">
 <div class="download">
   <h1>{% translate pagetitle %}</h1>
-  <h2>{% translate latestversion %} {{ site.DOWNLOAD_VERSION }}<a type="application/rss+xml" href="/en/rss/releases.rss"><img src="/img/icons/icon_rss.svg" alt="rss" class="rssicon"></a></h2>
+  <div class="inline-block">
+    <h2 id="latestversion">{% translate latestversion %} {{ site.DOWNLOAD_VERSION }}<a type="application/rss+xml" href="/en/rss/releases.rss"><img src="/img/icons/icon_rss.svg" alt="rss" class="rssicon"></a></h2>
+  </div>
```

Comment:
@harding This workaround kind of breaks down the layout for mobile users (see screenshot below), probably because the download button isn't shown. May I suggest you convert this title into a stylized DIV?

![capture du 2015-04-09 15 30 50](https://cloud.githubusercontent.com/assets/3578089/7075131/a621df16-decd-11e4-9af5-22cbbc981a72.png)